lectures/lake_model.md

```diff
@@ -0,0 +1,205 @@
+
```

Comment:
@jstac looks like this empty line needs to be removed for `jupytext` to understand the metadata